### ISSUE TITLE
Is possible make it compatible with SYSTRAN 6 OFFLINE ?

### ISSUE THREAD

**Issue by FabioManoel1977** [Sinflower/Translation-Aggregator#9]
Sorry bad english

I am getting terrible troubles with my internet connection, all the time disconnects. Today I get 8 hours without internet connection. Is making impossible to use online translators. I installed LEC, ATLAS and SYSTRAN 6, all offline translators, but looks like  SYSTRAN 6 OFFLINE not works. ATLAS and LEC worked fine, but, I wanted to compare 3 different translations.

Is possible make Translator Aggregator compatible with SYSTRAN 6 OFFLINE ?

Exist any other offline translator that I can use with translator aggregator ?

**Comment by Sinflower** [comment]
Please take a look at [this](https://amaenboda.wordpress.com/2012/03/26/an-aero-gamers-guide-to-using-atlas/6/#Systran) guide, it should explain how to setup TA to use the offline version of systran.

**Comment by FabioManoel1977** [comment]
Yes I did it, but for some unknow reason not works inside translator aggregator, Systran works fine outside. I guess TAHelper plugin not works on windows 10. Just be forever without change the message "SYSTRAN: translation will appear here"

**Comment by Sinflower** [comment]
I managed to get it to work in Windows 10.
I used the latest TAHelper from https://github.com/setsumi/TAHelper/releases
For it to work with TA you will also need to add the TAHplugin to TA (if not present create a folder called Plugins in the directory containing "Translation Aggregator.exe" and place the TAHplugin.dll found in the TAHelper folder there). Next, run TA, TAHelper and Systran toolbar, add a new window, disable automatic translation for that window and attach Systran to that window (using the + from TAHelper). Make sure you have TAHelper enabled (check "Enable" in the top left) and it should now translate all text you copy to the clipboard.
